### PR TITLE
clean unused scrapers

### PR DESCRIPTION
**Pull request by sglyon**
- wip on cleaning scrapers
- fix column names for TX vaccine demographics scraper
- update service name for AL vaccine
- remove engine="xlrd" (deprecated) from MI vaccine
- clear out init files
- isort
- black

### CODE REVIEW

**Comment by sglyon** [comment]
Hey @ghop02 this cleans house.

I deleted scrapers that we don't need anymore (only provide cases | deaths | tests | hospitalization with demographic data), reorganized the imports, fixed AL vaccine, FL vaccine, Mi vaccine and ran isort + black

File: scratch.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import asyncio
```
Comment (ghop02) [inline scratch.py:1]: Was this intentional?

**Review by ghop02 — APPROVED** [review]
This looks good to me!

File: scratch.py (hunk 1 above)
Comment (sglyon) [reply scratch.py:1]: whoops, now

**Pull request merged**